persistence/sql/migrations/20230908104443000000_change_client_pk.cockroach.up.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE hydra_client ALTER PRIMARY KEY USING COLUMNS (id, nid) USING HASH;
```

Comment:
Now that `id` can be user-set, this can lead to ID collisions. We need to ensure that all foreign key lookups consider the full primary key.